README.md

```diff
@@ -49,11 +49,10 @@ app$layout_set(
   htmlDiv(id = "outputID")
 )
 
-app$callback(
-  function(x = input("inputID"), y = input("inputID", "type")) {
+app$callback(output=list("outputID"), inputs=list(list("inputID", "type")), user_function=
```

Comment:
Is `user_function=` needed? Or can a user just pass in `function(value) {...}`?

Reply:
The way the `callback` method is currently defined, `user_function=` must be specified, or the following error message is returned:

``Error in assert_valid_callbacks(output, inputs, state, user_function) : 
  argument "user_function" is missing, with no default``